--- Apache-Session-1.53/Makefile.PL	Fri May 26 08:11:53 2000
+++ Apache-Session-1.53-p/Makefile.PL	Fri Jun 22 14:35:50 2001
@@ -5,6 +5,6 @@
 
   NAME => "Apache::Session",
   VERSION_FROM => "Session.pm",
-  PREREQ_PM => { MD5 => 0, Storable => 0 },
+  PREREQ_PM => { Digest::MD5 => 0, Storable => 0 },
   'dist' => { COMPRESS=>"gzip", SUFFIX=>"gz" }
 );
--- Apache-Session-1.53-p/Session/Generate/MD5.pm	Sat Sep  2 00:03:43 2000
+++ Apache-Session-1.53/Session/Generate/MD5.pm	Fri Jun 22 14:40:40 2001
@@ -11,7 +11,7 @@
 
 use strict;
 use vars qw($VERSION);
-use MD5;
+use Digest::MD5;
 
 $VERSION = '2.0';
 
